package.json

```diff
@@ -42,6 +42,7 @@
     "update:readme": "bash ./scripts/updateReadme.sh",
     "release": "np",
     "release:prepare": "echo 2Fabiola for the bigger and bolder",
+    "release-component:prepare": "node build",
     "release:testless": "np --yolo",
     "explode": "node exploder.js"
   },
```

Comment:
Ik mis precies nog scripts om de testen van de vl-ui-util te runnen?

Reply:
er zijn nog geen testen. die is @Pieter-Beckers aan het voorzien in ander ticket.